tests/layer_tests/pytorch_tests/test_all.py

```diff
@@ -72,8 +75,13 @@ def test_all_noparams(self, input_tensor, ie_device, precision, ir_version):
     @pytest.mark.precommit
     @pytest.mark.xfail(condition=platform.system() == 'Darwin' and platform.machine() == 'arm64',
                        reason='Ticket - 122715')
-    def test_all(self, input_tensor, keepdim, ie_device, precision, ir_version):
-        self.input_tensor = input_tensor
-        for dim in range(len(input_tensor.shape)):
-            self._test(aten_all(dim, keepdim), None, "aten::all", 
+    def test_all(self, input_shape, d_type, keepdim, ie_device, precision, ir_version):
+        if type(input_shape) is list:
+            self.input_tensor = np.random.randint(0, 2, input_shape, dtype=d_type)
```

Comment:
Can we avoid adding such ifs?

Reply:
we need to decide inside of test should we generate tensor data or it is already generated and we should use it as is.
 we can change the test arguments (for example, we can use dict), but it will change if condition only